build.py

```diff
@@ -571,6 +575,10 @@ def make_command(self, cmd):
             cmd = 'cmd /c ' + cmd
         return cmd
 
+    def add_unit_test_shortcuts(self):
```

Comment:
Can you fold this function into `add_run_test_builds`?

Reply:
Done.